.vscode/extensions.json

```diff
@@ -4,6 +4,7 @@
     "hashicorp.terraform",
     "ms-azuretools.vscode-docker",
     "dbaeumer.vscode-eslint",
-    "esbenp.prettier-vscode"
+    "esbenp.prettier-vscode",
+    "redhat.vscode-yaml"
   ]
 }
```

Comment:
A few other extensions that might be nice to include here:
- `bpruitt-goddard.mermaid-markdown-syntax-highlighting` ([link](https://marketplace.visualstudio.com/items?itemName=bpruitt-goddard.mermaid-markdown-syntax-highlighting)): Mermaid is [recommended in our ADR docs](https://github.com/usdigitalresponse/usdr-gost/tree/main/docs/decisions#how-to-diagrams-in-adrs), and is [used in `ADR 1`](https://github.com/usdigitalresponse/usdr-gost/blob/main/docs/decisions/0001-decision-record-process.md#process), so syntax highlighting support seems useful. 
- `bierner.markdown-mermaid` ([link](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-mermaid)): Similar to above, being able to preview mermaid diagrams feels all but necessary to effectively create a mermaid diagram.
- `github.vscode-github-actions` ([link](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions)): Super useful for editing our GitHub automations.
- `mikestead.dotenv` ([link](https://marketplace.visualstudio.com/items?itemName=mikestead.dotenv)): Helpful for folks who are less familiar with editing dotenv files.